packages/ui-library/src/components/forms/checkbox/index.stories.ts

```diff
@@ -220,8 +220,9 @@ export default {
       //Second Example
       // source: { code: '<blr-textarea label="Text Area LG" size="lg"></blr-textarea>',},
       description: {
-        component: `<Markdown>
-Checkbox represents two states: checked (selected) or unchecked (deselected), it  is usually accompanied by text labels that describe the option or action associated with the Checkbox.        
+        component: `<markdown>
+Checkbox represents two states: checked (selected) or unchecked (deselected), it  is usually accompanied by text labels that & describe the option or action associated with the Checkbox.        
```

Comment:
Now there is a "&" between that and describe. Please remove it so there is just one space between the two words.

Reply:
I also now see that we should write "text label" instead of "text labels", as the checkbox can only have one label and not multiple

Reply:
done